.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "es5",
+    "printWidth": 80,
+    "tabWidth": 2
+  }
+  
```

Comment:
You don't have to add this here. we already have `.prettierrc` in the `main` branch so remove it. And undo all the formatting changes.

Reply:
ok